packages/playwright-test/src/plugins/dockerPlugin.ts

```diff
@@ -33,20 +33,13 @@ export const dockerPlugin: TestRunnerPlugin = {
     await checkDockerEngineIsRunningOrDie();
     let info = await containerInfo();
     if (!info) {
-      print(colors.dim(`Starting docker container... `));
-      const time = Date.now();
-      info = await ensurePlaywrightContainerOrDie();
-      const deltaMs = (Date.now() - time);
-      println(colors.dim('Done in ' + (deltaMs / 1000).toFixed(1) + 's'));
-      println(colors.dim('The Docker container will keep running after tests finished.'));
-      println(colors.dim('Stop manually using:'));
-      println(colors.dim('    npx playwright docker stop'));
+      throw new Error('ERROR: please launch docker container separately!');
     }
     println(colors.dim(`View screen: ${info.vncSession}`));
     println('');
     process.env.PW_TEST_CONNECT_WS_ENDPOINT = info.wsEndpoint;
```

Comment:
- this will be `PLAYWRIGHT_ENDPOINT`, vscode will set it to it's own endpoint and docker plugin will not engage;
- vscode will set `reuse` header;
- `run-server --controlled-by-tether-client` to make server use global socks proxy for all clients;
- `ENV=SERVER_AUTO_DIE run-server` to auto-die after tether disconnects.